app/controllers/unattended_controller.rb

```diff
@@ -182,7 +182,7 @@ def set_content_type
 
   def load_template_vars
     # load the os family default variables
-    send "#{@host.os.pxe_type}_attributes"
+    send "#{@host.operatingsystem.pxe_type}_attributes"
```

Comment:
Would you suggest to create a new PR for this or merge it with this PR? Also what would you like more `host.os` or `host.operatingsystem`? I would prefer `host.operatingsystem`.